### PR TITLE
Subtract feature

### PR DESCRIPTION
**Pull request by WouterRuigrok**

### CODE REVIEW

File: src/service/subtract_service.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+
+def add(string):
+    arguments = string.split("-")
+    result = int(arguments[0]) - int(arguments[1])
```
Comment (paulbrandkvernelandmechatronics) [inline src/service/subtract_service.py:5]: Add try except for value error

File: src/service/calculation_service.py [hunk 2]
```diff
@@ -29,10 +30,12 @@ def equalpress(self):
         try:
 
             # evaluate the expression/calculation
-            if "/" not in self.expression:
-                total = str(eval(self.expression))
-            else:
+            if "/" in self.expression:
                 total = str(divide_service.divide(self.expression))
+            if "-" in self.expression:
```
Comment (paulbrandkvernelandmechatronics) [inline src/service/calculation_service.py:35]: use elif

File: src/service/subtract_service.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+
+def add(string):
```
Comment (paulbrandkvernelandmechatronics) [inline src/service/subtract_service.py:3]: rename to subtract

File: src/service/calculation_service.py [hunk 4]
```diff
@@ -1,6 +1,7 @@
 """Module for the CalculationService class"""
 from tkinter import StringVar
 import divide_service as divide_service
+import subtract_service as subtract_service
 
 
 class CalculationService:
```
Comment (paulbrandkvernelandmechatronics) [inline src/service/calculation_service.py:7]: Add documentation

File: src/service/subtract_service.py (hunk 3 above)
Comment (paulbrandkvernelandmechatronics) [inline src/service/subtract_service.py:3]: Add documentation

File: src/service/calculation_service.py [hunk 5]
```diff
@@ -29,10 +30,12 @@ def equalpress(self):
...
-            else:
+            if "/" in self.expression:
                 total = str(divide_service.divide(self.expression))
+            if "-" in self.expression:
+                total = str(subtract_service.subtract(self.expression))
+            else:
+                total = str(eval(self.expression))
 
             if float(total) > 10:
                 total = total + " <:o)"
```
Comment (paulbrandkvernelandmechatronics) [inline src/service/calculation_service.py:41]: Nice easteregg :D

**Comment by WouterRuigrok** [comment]
review gedaan

**Pull request merged**